pulsar-client/src/main/java/org/apache/pulsar/client/impl/ConsumerBuilderImpl.java

```diff
@@ -222,4 +223,10 @@ private ConsumerBuilderImpl(PulsarClientImpl client, ConsumerConfigurationData<T
         conf.setPatternAutoDiscoveryPeriod(periodInMinutes);
         return this;
     }
+
+	@Override
+	public ConsumerBuilder<T> subscriptionInitialPosition(SubscriptionInitialPosition subscriptionInitialPosition) {
+        conf.setSubscriptionInitialPosition(subscriptionInitialPosition);
```

Comment:
nit: wrong ident?

Reply:
it displays normally on my editors, both vim and vscode